cpp/simulations/abm_braunschweig.cpp

```diff
@@ -891,27 +891,27 @@ mio::abm::Simulation create_sampled_simulation(const std::string& input_file, co
     ScalarType exposed_prob = 0.005, infected_no_symptoms_prob = 0.001, infected_symptoms_prob = 0.001,
                recovered_prob = 0.0;
 
-    //Set global infection parameters (similar to infection parameters in SECIR model) and initialize the world
-    auto world = mio::abm::World(num_age_groups);
+    //Set global infection parameters (similar to infection parameters in SECIR model) and initialize the model
+    auto model = mio::abm::Model(num_age_groups);
 
-    set_parameters(world.parameters);
+    set_parameters(model.parameters);
 
-    // Create the world object from statistical data.
-    create_world_from_data(world, input_file, t0, max_num_persons);
-    world.use_migration_rules(false);
+    // Create the model object from statistical data.
+    create_model_from_data(model, input_file, t0, max_num_persons);
+    model.use_mobility_rules(false);
```

Comment:
One should maybe deactivate the mobility rules directly before one sets the trips?